applications/luci-app-tailscale/htdocs/luci-static/resources/view/tailscale/interface.js

```diff
@@ -0,0 +1,135 @@
+/* SPDX-License-Identifier: GPL-3.0-only
+ *
+ * Copyright (C) 2022 ImmortalWrt.org
+ * Copyright (C) 2024 asvow
+ */
+
+'use strict';
+'require dom';
+'require fs';
+'require poll';
+'require ui';
+'require view';
+
+return view.extend({
+	load: function() {
+		return fs.exec('/sbin/ifconfig').then(function(res) {
+			if (res.code !== 0 || !res.stdout || res.stdout.trim() === '') {
+				ui.addNotification(null, E('p', {}, _('Unable to get interface info: %s.').format(res.message)));
+				return '';
+			}
+
+			var interfaces = res.stdout.match(/tailscale[0-9]+/g);
+			if (!interfaces || interfaces.length === 0)
+				return 'No interface online.';
+
+			var promises = interfaces.map(function(name) {
+				return fs.exec('/sbin/ifconfig', [name]);
+			});
+
+			return Promise.all(promises).then(function(results) {
+				var data = results.map(function(res, index) {
+					if (res.code !== 0 || !res.stdout || res.stdout.trim() === '') {
+						ui.addNotification(null, E('p', {}, _('Unable to get interface %s info: %s.').format(interfaces[index], res.message)));
+						return null;
+					}
+					return {
+						name: interfaces[index],
+						stdout: res.stdout.trim()
+					};
+				}).filter(Boolean);
+
+				return data.map(function(info) {
+					var lines = info.stdout.split('\n');
+					var parsedInfo = {
+						name: info.name
+					};
+
+					lines.forEach(function(line) {
+						if (line.includes('inet addr:')) {
+							parsedInfo.ipv4 = line.split('inet addr:')[1].trim().split(' ')[0];
+						} else if (line.includes('inet6 addr:')) {
+							parsedInfo.ipv6 = line.split('inet6 addr:')[1].trim().split('/')[0];
+						} else if (line.includes('MTU:')) {
+							parsedInfo.mtu = line.split('MTU:')[1].trim().split(' ')[0];
+						} else if (line.includes('RX bytes:')) {
+							var rxMatch = line.match(/RX bytes:\d+ \(([\d.]+\s*[a-zA-Z]+)\)/);
+							if (rxMatch && rxMatch[1]) {
+								parsedInfo.rxBytes = rxMatch[1];
+							}
+							var txMatch = line.match(/TX bytes:\d+ \(([\d.]+\s*[a-zA-Z]+)\)/);
+							if (txMatch && txMatch[1]) {
+								parsedInfo.txBytes = txMatch[1];
+							}
+						}
+					});
```

Comment:
Are you certain this does what you want? Parsedinfo will always correspond to the last parsed entry. 

I dislike parsing ifconfig output. If possible, change to the `ip` command. `ip -s ad` should do what you need, but is not available in busybox. So you need to add a dep for `+ip`. 

Further, you can improve this by: `ip -s -j ad` and piping to `jsonfilter` to get exactly the fields you need for exactly the interface you want. It can also get all of the interfaces at the same time.